test/test_matcher.py

```diff
@@ -16,7 +17,14 @@ def aws_credentials():
     os.environ['AWS_SECURITY_TOKEN'] = 'testing'
     os.environ['AWS_SESSION_TOKEN'] = 'testing'
 
+
 @pytest.fixture(scope='function')
+def kms(aws_credentials):
+    with mock_kms():
+        yield boto3.client('kms', region_name='eu-west-2')
+
+
+@pytest.fixture(scope='function')  #
```

Comment:
Is the hash at the end of the line a typo?

Reply:
Yep, I've taken it out.